roles/haproxy/tasks/main.yml

```diff
@@ -1,7 +1,6 @@
 ---
 - name: Install haproxy
   action: "{{ ansible_pkg_mgr }} name=haproxy state=present"
-  when: not openshift.common.is_containerized | bool
```

Comment:
I'd just leave this for now. I'm doing a sweep where I'll be differentiating between is_containerized and is_atomic and I'll get this there unless this is critical to fixing the docker config on lb hosts.